analysis/analyzer.py

```diff
@@ -46,12 +51,67 @@
 ]
 
 
+class NetworkResult:
+  """Aggregates the analysis results for a specific network model."""
+
+  def __init__(self, name):
+    self.name = name
+    self.latency_distribution = distribution.Distribution(
+        distribution.LinearBucketer(5))
+    self.cost_distribution = distribution.Distribution(
+        distribution.LinearBucketer(5))
```

Comment:
Can you just collect the actual values and then use pythons existing strong support for analysis?

For example, dump into a pandas DataFrame and qcut it.

Reply:
Makes sense, however for now I'll probably just leave this as is and upgrade it to use pandas later on if we end up needing more advanced functionality.